dts/bindings/clock/nuvoton,npcx-pcc.yaml

```diff
@@ -11,12 +26,132 @@ properties:
     reg:
         required: true
 
+    clock-frequency:
+        required: true
+        type: int
+        description: |
+          default frequency in Hz for HFCG output clock (OFMCLK).
+        enum:
+          - 100000000 # 100 MHz
+          - 96000000 # 96 MHz
+          - 90000000 # 90 MHz
+          - 80000000 # 80 MHz
+          - 66000000 # 66 MHz
+          - 50000000 # 50 MHz
+          - 48000000 # 48 MHz
+          - 40000000 # 40 MHz
+          - 33000000 # 33 MHz
```

Comment:
If you move these comments into the description itself, it will show up in the bindings index:

https://docs.zephyrproject.org/latest/reference/devicetree/bindings.html

I think that is preferable to forcing people to find the binding source code. Similar comments apply to the rest of the properties here.

Reply:
Got it. I will move them to descriptions.

Reply:
Done.